src/app/courses/progress-courses/courses-progress.scss

```
@@ -1,7 +1,463 @@
+      min-height: 0;
+    }
+
+    .progress-table {
+      width: 100%;
+      border-collapse: separate;
+
+      th.mat-header-cell {
+        font-weight: 600;
+        color: #444;
+
+        font-size: 0.85rem;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)